modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -269,6 +269,15 @@ bond=bond0:em1,em2:mode=active-backup
 vlan=bond0.100:bond0
 ----
 
```

Comment:
It would be useful to add an`id` metadata entry here. E.g. something like:
```suggestion
[id="bonding-multiple-network-interfaces-with-mtu_{context}"]
```

Reply:
Done.